test/tpu/cloudbuild.yaml

```diff
@@ -0,0 +1,60 @@
+substitutions:
+  # GCS location to write temporary checkpoints and summaries.
+  _GCS_BUCKET: 'gs://pytorch-xla-tpu-ci-tests/'
+  # Name of GKE cluster in which to run Job.
+  _CLUSTER_NAME: 'tpu-cluster'
+  # Location of GKE cluster.
+  _CLUSTER_ZONE: 'europe-west4-a'
+steps:
+# We only need to update submodules in triggers. User must update submodule
+# before local runs because .git is not present.
+- name: 'alpine/git'
+  entrypoint: sh
+  args:
+  - -c
+  - |
+    git submodule update --init || echo No git repository found
+- name: 'docker'
+  id: build-image
+  args: [
+    'build',
+    '.',
+    '-f',
+    'docker/experimental/Dockerfile',
+    '-t',
+    'gcr.io/$PROJECT_ID/pytorch-xla-test:$BUILD_ID',
+    '--build-arg',
+    'tpuvm=1',
+    '--build-arg',
+    'bazel_jobs=16'
+  ]
+- name: 'docker'
+  id: push-image
+  waitFor:
+  - build-image
+  args: ['push', 'gcr.io/$PROJECT_ID/pytorch-xla-test:$BUILD_ID']
+- name: 'google/cloud-sdk'
+  id: create-job
+  waitFor:
+  - push-image
+  entrypoint: bash
+  args:
+  - -c
+  - |
+    set -u
+    set -e
+    set -x
+
+    apt-get update
+    apt-get -y install gettext
+
+    gcloud container clusters get-credentials --zone $_CLUSTER_ZONE $_CLUSTER_NAME
+    export BUILD_ID=$BUILD_ID
```

Comment:
Add a comment that you're just adding these variables to the shell environment, since it's probably not clear at first to someone who doesn't know GCB that well

Reply:
Done